translation.yml

```diff
@@ -0,0 +1,15 @@
+source_language: en
+target_languages: [bg-BG, cs, da, de, el, es, fi, fr, hr-HR, hu, id, it, ja, ko, lt-LT, nb, nl, pl, pt-BR, pt-PT, ro-RO, ru, sk-SK, sl-SI, sv, th, tr, vi, zh-CN, zh-TW]
+non_blocking_languages: []
+components:
+  - name: buyer
+    audience: buyer
+    scheme: 'online-store-theme'
```

Comment:
What does `scheme` do?

Reply:
From the docs: `A schema is the way that a file should be parsed at an application level.`

Reply:
Thanks for the context 👍